src/main/cacheManager.ts

```diff
@@ -21,7 +22,7 @@ export function setRedisCredentials(): any {
 
 const logger = Logger.getLogger('app');
 
-let redisClient;
+export let redisClient;
```

Comment:
module.exports below could be removed too since it is aready marked as export here

Reply:
Good spot, updated